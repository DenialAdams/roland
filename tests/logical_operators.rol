proc f_false() -> bool {
    print("f_false");
    return false;
}

proc f_true() -> bool {
    print("f_true");
    return true;
}

proc main() {
    let result = f_true() or f_false();
    print_bool(result);
    print("---");
    result = f_false() and f_true();
    print_bool(result);
}
