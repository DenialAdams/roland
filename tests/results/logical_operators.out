f_true
true
---
f_false
false
